cmds/loop.js

```diff
@@ -14,15 +14,15 @@ module.exports = {
       if (member.voice.channel.id !== client.music.playing[guild.id].channel.id) return error("The bot is in a different VC");
 
       let buttonRow = new MessageActionRow()
-      let enable = new MessageButton().setLabel("Enable").setStyle("green").setID(`enableLoop`),
-        disable = new MessageButton().setLabel("Disable").setStyle("red").setID(`disableLoop`),
-        cancel = new MessageButton().setLabel("Cancel").setStyle("red").setID(`loopCancel`);
+      let enable = new MessageButton().setLabel("Single Loop").setStyle("blurple").setID(`enableLoop`),
+        disable = new MessageButton().setLabel("Queue Loop").setStyle("blurple").setID(`enableQueueLoop`),
+        cancel = new MessageButton().setLabel("Disable").setStyle("red").setID(`disableLoop`);
       buttonRow.addComponent(enable)
       buttonRow.addComponent(disable)
       buttonRow.addComponent(cancel)
       
       var msg = await respond({
-        content: "• Turn the loop system on/off.",
+        content: "• Turn the loop system queue type/single type/off.",
```

Comment:
Do it with `expectedArgs` choices

Reply:
its on your side to change the whole bot to the newest version of gcommands...